.storybook/preview.js

```diff
@@ -18,9 +20,13 @@ export const decorators = [
     return (
       <NextIntlClientProvider messages={getMessages(locale)} locale={locale}>
         <MUIThemeProvider theme={materialTheme}>
-          <ThemeProvider theme={materialTheme}>
-            <Story />
-          </ThemeProvider>
+          <TenantProvider>
+            <UserPropsProvider>
+              <ThemeProvider theme={materialTheme}>
+                <Story />
+              </ThemeProvider>
+            </UserPropsProvider>
+          </TenantProvider>
```

Comment:
As we would always have the default tenant, and there would be no user for storybook, add a comment explaining why `TenantProvider` and `UserPropsProvider` are added here.